.env.example

```diff
@@ -44,4 +44,11 @@ OPENID_JWS_ALG=
 OPENID_RSA_PRIVATE_KEY=
 # openssl rsa -in private_key.pem -pubout -out public_key.pem
 # cat public_key.pem | base64
-OPENID_RSA_PUBLIC_KEY= 
\ No newline at end of file
+OPENID_RSA_PUBLIC_KEY= 
+
+# You can use `openssl req -x509 -newkey rsa:2048 -keyout key.pem -out public.crt -sha256 -days 365000 -nodes` to generate one
+# Base64 encoded value of public key `cat public.crt | base64`
+JACKSON_PUBLIC_KEY=
+
+# Base64 encoded value of private key `cat key.pem | base64`
+JACKSON_PRIVATE_KEY=
```

Comment:
@deepakprabhakara Any suggestion on what key names? At the moment, I have added `JACKSON_PUBLIC_KEY` and `JACKSON_PRIVATE_KEY`.